index.rst

```diff
@@ -4,9 +4,326 @@ Monitor coupling in M1M3 VMS system and TMA drives together
+it is easy to retrieve the corresponding TMA position in azimuth. The result is shown on the following figure.
+
+.. figure:: /_static/images/VMS-peaks-2023-12-21-558.png
+   :name: VMS-peaks-2023-12-21-558-2023-12-21-558
+   :target: _images/VMS-peaks-2023-12-21-558.png
+   :alt: VMS signal from accelerometer 1 along the x axis filtered and rebinned in order to be able to locate the glitches (corresponding to the peaks in the plot).
+
+   VMS signal from accelerometer 1 along the x axis filtered and rebinned in order to be able to locate the glitches (corresponding to the peaks in the plot).
+
+On this particular slew we find 5 glitches corrresponding to the following azimuth angles: [-72.5 -70.3 -54.7  66.3  68.7] degrees. Note that these positions are not very accurate as the glitches 
+have a certain duration and the definition of their time of occurence is ambiguous (at least with the method used).
+
+We have analyzed several large amplitude slews with the TMA rotating in both directions and got the following locations for the glitches:
+
+* Seqnum 91:  [-72.5 -70.3 -54.7  66.3  68.7] degrees
+* Seqnum 92:  [ 82.8  80.2  65.4 -55.  -57.2 -74.6 -77.6] degrees
+* Seqnum 94:  [ 82.8  80.8  65.8 -54.8 -57.  -74.4 -76.4] degrees
+* SeqNum 137: [-70.7 -60.9 -53.9  68.3  86.1] degrees
+
+Taking into account the uncertainty on the glitches localization we see that they tend to appear close to the same TMA azimuth angles:  ~-75 deg / -55 deg / +65 deg / +80 deg. The cause of
+these vibrations is unknown.
+
+The next figure summarize the results of the PSD analysis on the 3 axes of the 3 VMS accelerometers.
+
+.. figure:: /_static/images/psd-2023-12-21-558-.png
+   :name: psd-2023-12-21-558
+   :target: _images/psd-2023-12-21-558-.png
+   :alt: PSD (acceleration) for the 3 axes of the 3 VMS accelerometers. The detected frequency peaks are ndicated by a red cross.
+
+   PSD (acceleration) for the 3 axes of the 3 VMS accelerometers. The detected frequency peaks are ndicated by a red cross.
+
+
+The detected peak frequencies in Hz for the different accelerometers / axes are the following:
+
+* Accel 1 - x - Frq: [1.57, 7.78, 14.01, 15.56, 19.05, 31.11, 41.98, 42.77]
+* Accel 1 - y - Frq: [1.53, 7.78, 14.01, 15.56, 16.33, 17.38, 19.46, 31.11]
+* Accel 1 - z - Frq: [1.53, 14.01, 15.56]
+* Accel 2 - x - Frq: [14.01, 15.56, 46.67]
+* Accel 2 - y - Frq: [14.01, 15.56]
+* Accel 2 - z - Frq: [14.01, 15.52, 19.48]
+* Accel 3 - x - Frq: [1.57, 14.01, 15.56, 18.58, 19.46, 31.11, 41.98, 46.67]
+* Accel 3 - y - Frq: [1.55, 14.01, 15.56, 19.01]
+* Accel 3 - z - Frq: [1.55, 14.01, 15.56]
+
+The very low frequency around 1.6 Hz is very likely associated to the acceleration and deceleration phases.
+
+
+Elevation slews
+---------------
+
+Finding data period with large amplitude elevation only slews is more difficult tahn for azimuth slew. We selected the following one: 
```

Comment:
tahn --> than

Reply:
Is this a 'typical' example?

Reply:
I wasn't able to find enough large elevation slews where the TMA is not moving in azimuth to make statistics, so it is a particular event.